src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random RANDOM = new Random();
+    private static final int FIGURE_COUNT = 5;
+    private static final int MIN_LENGTH = 2;
+    private static final int MAX_LENGTH = 10;
+    private static final int DEAFULT_FIGURE_RADIUS = 10;

```

Comment:
There is a typo in the constant name 'DEAFULT_FIGURE_RADIUS'. It should be 'DEFAULT_FIGURE_RADIUS'.